docs/deployment/universal-profile.md

```diff
@@ -433,53 +431,40 @@ lspFactory.UniversalProfile.deploy({...}, {
 })
 ```
 
-If the `options` object is provided, it will override the `options` object passed during the instantiation of the LSPFactory.
+If the `ipfsClientOptions` object is provided, it will override the `ipfsClientOptions` object passed during the instantiation of the LSPFactory.
 
 ### Reactive Deployment
 
-The LSPFactory uses [RxJS](https://rxjs.dev/) library to deploy contracts. Developers can leverage the process to achieve reactive deployment of Universal Profiles.
-When deploying a Universal Profile, pass the `deployReactive` flag inside the `contractDeploymentOptions` object to receive an [RxJS](https://rxjs.dev/) Observable, which will emit events as your contract is deployed.
+LSPFactory uses [RxJS](https://rxjs.dev/) to deploy smart contracts. This can be leveraged for reactive deployment of Universal Profiles. [Read more here](../getting-started.md#reactive-deployment).
 
-```typescript
-const universalProfileDeploymentObservable = lspFactory.UniversalProfile.deploy({...}, {
-    deployReactive: true
-  }
-);
+When `deployReactive` is set to `true`, an [RxJS Observable](https://rxjs.dev/guide/observable) will be returned which will emit events as the deployment progresses.
```

Comment:
Suggestion:


When `deployReactive` is set to `true`, an [RxJS Observable](https://rxjs.dev/guide/observable) will be returned. This observable will emit events as the deployment progresses.